justfile

```diff
@@ -0,0 +1,25 @@
+alias b := build
+alias c := check
+alias f := fmt
+alias t := test
+
+_default:
+   @just --list
+
+# Build the project
+build:
+   cargo build
+
+# Check code: formatting, compilation, linting, and commit signature
+check:
+   cargo +nightly fmt --all -- --check
```

Comment:
I don't use `rustup` and this would fail.
Can you please remove the `+nightly bound`